.github/workflows/sonarqube-build.yml

```diff
@@ -21,3 +21,10 @@ jobs:
       #   timeout-minutes: 5
       #   env:
       #     SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+      
+      # Temporary cleanup solution to prevent failing of next job.
+      - name: Clean up GitHub workspace
+        if: always()
```

Comment:
Is that needed?

Reply:
Unfortunately yes. There is a bug in self-hosted github runners where files are created that should be deleted the next time a workflow is run, but then the permissions are missing. The solution here is unfortunately the only sensible one at the moment without manually deleting the corresponding permissions each time after the action has been